apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -221,6 +223,8 @@ class ScriptEditor extends React.Component {
       family_name: this.state.familyName,
       is_course: this.state.isCourse,
       show_calendar: this.state.showCalendar,
+      weekly_instructional_minutes:
```

Comment:
I think we may just not want leave this property out of the update if it does not have a value.  Similar to stage_descriptions below.